docs/app/js/css-api-table.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+  angular.module('docsApp')
+    .directive('docsCssApiTable', DocsCssApiTableDirective)
+    .directive('docsCssSelector', DocsCssSelectorDirective);
+
+  function DocsCssApiTableDirective() {
+    return {
+      restrict: 'E',
+      transclude: true,
+
+      bindToController: true,
+      controller: DocsCssApiTableController,
+      controllerAs: '$ctrl',
+
+      scope: {},
+
+      template:
+      '<table class="md-api-table md-css-table">' +
+      '  <thead>' +
+      '    <tr><th>Available Selectors</th></tr>' +
+      '  </thead>' +
+      '  <tbody ng-transclude>' +
+      '  </tbody>' +
+      '</table>'
+    }
+  }
+
+  function DocsCssApiTableController() {
```

Comment:
Nit: Why are there controllers if they aren't used? Same for the selector directive.